hs/src/Reach/Backend/JS.hs

```diff
@@ -456,20 +456,20 @@ jsExpr = \case
             [ ("amt", zero')
             , ("tok", tok')
             ]
-  DLE_TokenAccepted _ acc tok -> do
+  DLE_TokenAccepted _ addr tok -> do
     (ctxt_mode <$> ask) >>= \case
-      JM_Backend -> do
-        -- TODO - This is wrong, but I'm still not sure what to do here.
-        return $ "false"
+      JM_Backend -> return "true"
       JM_View -> impossible "view canReceive"
       JM_Simulate -> do
-        acc' <- jsArg acc
+        addr' <- jsArg addr
         tok' <- jsArg tok
+        let simTxn = jsSimTxn "tokenAccepted" $ [("tok", tok'), ("addr", addr')]
+        -- TODO - this doesn't work because addr' is a JS string, so it doesn't have a tokenAccepted method.  How do I transform it into an address here?  Or am I going about this the wrong way?
+        let accepted = "await" <+> jsApply (addr' <> ".tokenAccepted") [tok']
```

Comment:
Something that often works is to add a new function, `simTokenAccepted`, to the interface and have it be the runtime code that calls the appropriate helper. But not necessarily... this is kind of a taste/figure-out-what-is-already there question.